src/app/project-sets/columns.tsx

```diff
@@ -0,0 +1,46 @@
+"use client"
+
+import {type ColumnDef} from "@tanstack/table-core"
+import {Checkbox} from "@/components/ui/checkbox"
+import {DataTableColumnHeader} from "@/components/ui/data-table-column-header"
+
+export type ProjectSet = {
```

Comment:
idk if this type should be here

Reply:
either it shouldn't be defined here or it should be named more specific to it being just for this table